app/renderer/js/main.js

```diff
@@ -87,7 +90,12 @@ class ServerManagerView {
 			showNotification: true,
 			betaUpdate: false,
 			silent: false,
-			lastActiveTab: 0
+			lastActiveTab: 0,
+			dnd: false,
+			oldSettings: {
```

Comment:
@cPhost, @jainaman224 can we make it more readable? I was thinking to use something like "dndState", "dndOldState", "dndPreviousSettings".

Reply:
Yeah, we can this make's it more clear. (Not that it was not clear before, but yeah this is great)

Reply:
I have changed it to `dndPreviousSettings`.